presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -8301,7 +8301,7 @@ public void testLastValueIgnoreNulls()
     @Test
     public void testLargeBytecode()
     {
-        StringBuilder stringBuilder = new StringBuilder("SELECT x FROM (SELECT orderkey x, custkey y from orders limit 10) WHERE CASE");
+        StringBuilder stringBuilder = new StringBuilder("SELECT x FROM (SELECT orderkey x, custkey y from orders limit 10) WHERE CASE true ");
```

Comment:
Why do we need to change this? Would this not trigger bytecode too large anymore?

Reply:
So the code now stack overflows in the bytecode generator. So part2 of this change will also fix that.